Replicator/DatabaseCookies.cc

```diff
@@ -42,8 +42,11 @@ namespace litecore { namespace repl {
 
 
     void DatabaseCookies::saveChanges() {
-        if (!_store->changed())
+        if (!_store->changed()) {
+            WriteVerbose("Cookie Store unchanged, skipping save...");
```

Comment:
This doesn't seem worth logging.

Reply:
Probably not, I do remember going a bit crazy toward the end.